mars-form-app/src/components/Form.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import countries from './countries'
+import '../components/Form.css'
+// import axios from 'axios';
+
+
+// import { render } from 'react-dom';
+
+class Form extends React.Component {
+    state = {
+        name: "",
+        date: new Date().toLocaleDateString(),
+        country: "United States",
+        diet: "vegan",
+        body: "",
+        q1: "yes",
+        q2: "Unmarried",
+        q3: "defeat",
+        q4: ""
+    }
+
+    handleChange = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault();
+        let { name, date, country, diet, body, q1, q2, q3 } = this.state
+
+        console.log("this is the handleSubmit" + {
+            fullname: name,
+            dob: date,
+            origin: country,
+            food: diet,
+            summary: body,
+            breath: q1,
+            married: q2,
+            reaction: q3
+        })
+    }
```

Comment:
the + in your console log is not great. an object plus a string gives [object object]. Instead use a comma as console log takes in multiple arguments.